lib/p2p/Host.ts

```diff
@@ -10,6 +10,10 @@ class Host {
     return this.instance.id;
   }
 
+  get pubKey(): string {
```

Comment:
We should explicitly declare this as `public` (and the `get id()` above as well if you don't mind). I guess the compiler doesn't enforce explicitly visibility on getters, but it's nice to be consistent anyway.